apps/andi/lib/andi_web/live/ingestion_live_view/edit_ingestion_live_view.ex

```diff
@@ -132,6 +164,23 @@ defmodule AndiWeb.IngestionLiveView.EditIngestionLiveView do
     {:noreply, assign(socket, page_error: true, save_success: false)}
   end
 
+  def handle_info(
+        %{topic: topic, event: event, payload: payload}, socket) do
```

Comment:
I found these catch_all handlers to be helpful. It logs the info and then reassigns the current state, allowing the live_view to not crash when unexpected events come through, but still logging them.

Opinions? Are there better ways to log besides IO.inspect? This portion isn't critical to the refactor.

Reply:
I think this is great, I'm constantly putting these in while developing locally, they should ideally never fire, so calling them out in pod logs seems like a good tactic to reduce the occurrences of them happening.

This will make instances of this much easier to fix, and reduce the crashes that caused the "port2" bugs in the first place.